packages/astro/src/core/build/index.ts

```diff
@@ -142,7 +142,11 @@ class AstroBuilder {
         target: 'es2020', // must match an esbuild target
       },
       plugins: [
-        rollupPluginHTML({ input, extractAssets: false }) as any, // "any" needed for CI; also we don’t need typedefs for this anyway
+        rollupPluginHTML({
+          rootDir: viteConfig.root,
```

Comment:
This was the other fix: `@web/rollup-plugin-html` needs `rootDir` for Windows. Otherwise it tries to guess (which for some reason works on unix but fails miserably on Windows).